docs/user-manual.md

```diff
@@ -0,0 +1,471 @@
+
+</details>
+
+<details>
+
+<summary>Trades</summary>
+
+> - Shows your trades. Your most recent trades will be retrieved by default, but a timestamp can be used to retrieve time-specific data. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+> - Shows candles, provides OCHL (Open, Close, High, Low) and volume data for the specified funding currency or trading pair. This section provides optional functionality to sync certain pairs and time period. Contain sections `Date Range Selection`, `Filters`, `Query Button`, `Refresh Button`, `Sync Preferences`, `Chart`
+
+</details>
+
+<details>
+
+<summary>Orders</summary>
+
+> Shows historic closed or cancelled orders. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Movements</summary>
+
+> Shows your past deposits/withdrawals. Currency can be specified to retrieve movements specific to that currency. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Positions</summary>
+
+> - Shows data on past positions. Timestamps can be used to retrieve results for a specific time period. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+> - Shows an audit of your positions by click on position ID. Contain sections `Date Range Selection`, `Refresh Button`, `Table`
+> - Shows your active positions. Timestamps can be used to retrieve results for a specific time period. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Wallets</summary>
+
+> Shows account wallet balances. Contain sections `End Time Selection`, `Query Button`, `Refresh Button`, `Tables`
+
+</details>
+
+<details>
+
+<summary>Funding Bids & Offers</summary>
+
+> Shows past inactive funding offers. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Funding Loans (Unused)</summary>
+
+> Shows inactive funds not used in positions. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Funding Credit (Used)</summary>
+
+> Shows inactive funds used in positions. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Funding Earnings</summary>
+
+> Shows your past ledger entries for margin/swap/interest payment. Most recent entries are returned by default, but a timestamp can be used to retrieve time-specific data. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
```

Comment:
Not sure to understand this: `margin/swap/interest payment` i think that might be better to change to `interest earnings`